evm_arithmetization/src/fixed_recursive_verifier.rs

```diff
@@ -786,10 +787,19 @@ where
         all_stark: &AllStark<F, D>,
         degree_bits_ranges: &[Range<usize>; NUM_TABLES],
         stark_config: &StarkConfig,
+        shrinking_circuit_config: Option<&CircuitConfig>,
+        recursion_circuit_config: Option<&CircuitConfig>,
+        threshold_degree_bits: Option<usize>,
```

Comment:
nit: quid of grouping this into a `RecursionParams` struct or similar?

Reply:
Sure, I will work on it in a separate PR.